dask/array/tests/test_sparse.py

```diff
@@ -125,56 +127,6 @@ def test_tensordot():
     )
 
 
-@pytest.mark.xfail(reason="upstream change", strict=False)
```

Comment:
These tests were xfailed over four years ago -- it doesn't seem that `sparse.COO` is going to support mixed concatenation with numpy ndarrays, so I don't see the point in keeping these around